templates/problem/voting-controls.html

```diff
@@ -0,0 +1,206 @@
+<style>
+    .vote-form {
+        background-color: rgb(255,255,255);
+        padding: 40px;
+        border-radius: 25px;
+        justify-content: center;
+        align-items: center;
+        margin: auto;
+        text-align: center;
+    }
+
+    .vote-form-info {
+        font-size: 24px;
+        font-weight: bold;
+        padding:10px
+    }
+
+    .vote-form-value {
+        font-size: 24px;
+    }
+
+    .vote-form-text {
+        font-size: 18px;
+    }
+
+    .voting-form-error {
+        font-size: 18px;
+        color: red;
+    }
+
+    .vote-button {
+        float:right;
+    }
+
+    .form-button {
+        cursor: pointer;
+    }
+</style>
+
+{% if not request.in_contest %}
+    <hr style="padding-top: 0.1em">
+{% endif %}
+
+{% include 'lightbox.html' %}
+
+{% if can_vote %}
+    <div class="vote-form" id="id_vote_form_box">
+        <div id="id_has_voted_prompt">
+            <span class="vote-form-info">{{ _('Current Vote:') }}</span>
+            <span id="id_current_vote_value" class="vote-form-value"></span>
+            <br/>
+            <br/>
+            <span class="vote-form-text">{{  _('Change your vote below!') }}</span>
+            <br/>
+            <br/>
+        </div>
+        <div id="id_has_not_voted_prompt">
+            <span class="vote-form-text">{{ _("Cast your vote on this problem's weighting below!") }}</span>
+            <br/>
+            <br/>
+        </div>
+        <form id="id_vote_form">
+            {% csrf_token %}
+            <span class="vote-form-info">{{ _('Points') }}</span>
+            <input id="id_points" class="vote-form-value" type="number" step="1"
+                   min="{{ min_possible_vote }}" max="{{ max_possible_vote }}" name="points">
+            <br/>
+            <div id="id_points_error_box">
+                <br/>
+                <span id="id_points_error" class="voting-form-error"></span>
+                <br/>
+            </div>
+            <br/>
+            <span class="vote-form-info">{{ _('Note') }}</span>
+            <br/>
+            <textarea id="id_note" class="vote-form-text" name="note" rows="6" cols="80"></textarea>
+            <br/>
+            <div id="id_note_error_box">
+                <br/>
+                <span id="id_note_error" class="voting-form-error"></span>
+                <br/>
+            </div>
+            <br/>
+            <input type="submit" id="id_vote_form_submit_button" class="vote-button" value="{{ _('Vote') }}">
+            <input type="hidden" name="vote_confirmation"/>
+        </form>
+    </div>
+
+    <a class="form-button" id="id_vote_button"></a>
+
+    <form id="id_delete_vote_form">
+        {% csrf_token %}
+        <a class="form-button" id="id_delete_vote_button">{{ _('Delete vote') }}</a>
+    </form>
+    {% if vote is not None %}
```

Comment:
Pretty sure it should be `none` in Jinja.

Reply:
Oh ok, the server wasn't up last night so I didn't get to test it yet, so None probably doesn't work.

Reply:
For posterity, both now work ([see the "Note" here](https://jinja.palletsprojects.com/en/3.0.x/templates/#expressions)), though lowercase is recommended.